corehq/apps/data_analytics/models.py

```diff
@@ -42,9 +42,10 @@ class MALTRow(models.Model):
     use_threshold = models.PositiveSmallIntegerField(default=DEFAULT_PERFORMANCE_THRESHOLD)
     experienced_threshold = models.PositiveSmallIntegerField(default=DEFAULT_EXPERIENCED_THRESHOLD)
 
-    # this is an inclusive date representing the end of the window when the malt was last run.
-    # See ./malt_generator.py for implementation details.
-    last_run_enddate = models.DateField(default=None, blank=True, null=True)
+    # represents the end date for the window of form submissions used to last calculate the MALTRow
```

Comment:
Is this different from the _"submission date of the newest form used to calculate the MALTRow"_?

Reply:
Nope pretty much the same thing. It would be a bit misleading to say it always represents the submission date of the newest form used in the calculation because that isn't where the value is coming from necessarily. I still primarily view this as a window where the start is the start of the month, and the end is `last_run_end_date`. It does feel like I'm missing something that would simplify this though.

Reply:
Is there a clearer way to describe the true "source" of the value? My understanding is that this date is the result of `get_last_form_submission_received(domain)`, maybe it's not that simple?